libs/log/logger.go

```diff
@@ -25,6 +26,7 @@ func log(logger *slog.Logger, ctx context.Context, level slog.Level, msg string)
 	// skip [runtime.Callers, this function, this function's caller].
 	runtime.Callers(3, pcs[:])
 	r := slog.NewRecord(time.Now(), level, msg, pcs[0])
+	r.AddAttrs(slog.Int("pid", os.Getpid()))
```

Comment:
You can cache the `slog.Attr`. I propose adding a `pidAttr` function (can be separate file because it is used from 2 places) that uses `sync.Once` to initialize a global that contains it. The PID doesn't change once a process has started, so it's not necessary to make the system call more than once.

Reply:
After reading the code and the docs, I've moved to pid attribute to the `cmd/root/logger::initializeContext`. With that change we aren't limited to using only our own `libs/log/logger` wrappers, but can also use logger instances from the context (as we do in the `cmd/root`)